trialfun/ft_trialfun_neuromagSTI016fix.m

```diff
@@ -0,0 +1,90 @@
+function [trl, event] = ft_trialfun_neuromagSTI016fix(cfg)
```

Comment:
I am not so happy with the name of the function; I wonder whether people will find it this way. I think the purpose is not to fix something, but to work as a general trial function for all neuromag data (regardless of whether STI016 is broken or not). Let me try some others ...

- ft_trialfun_neuromag
- ft_trialfun_triux
- ft_trialfun_STI016